docs/lro-provisioning-state-specified.md

```diff
@@ -14,12 +14,15 @@ ARM OpenAPI(swagger) specs
 
 ## Output Message
 
-`${code} response in long running ${verb} operation is missing ProvisioningState property. A LRO PUT and PATCH operations response schema must have ProvisioningState specified.
+${code} response schema in long running PATCH operation is missing ProvisioningState property. A LRO PATCH operations 200 response schema must have ProvisioningState specified.
+${code} response schema in long running PUT operation is missing ProvisioningState property. A LRO PUT operations response schema must have ProvisioningState specified.
+200 response schema in long running DELETE operation is missing ProvisioningState property. A LRO DELETE operations 200 response schema must have ProvisioningState specified.
 
 ## Description
 
-This is a rule introduced to validate if a LRO PUT and PATCH operations response schema has "ProvisioningState" property specified.
+This is a rule introduced to validate if a LRO PUT, PATCH and DELETE operations response schema has "ProvisioningState" property specified.
 
 ## How to fix the violation
 
-For a LRO PUT and PATCH operations, add "ProvisioningState" property to the response schema.
+For a LRO PUT add "ProvisioningState" property to the response schema.
```

Comment:
For an LRO PUT add "ProvisioningState" property to the response schema of 200 and 201 status codes.
For an LRO PATCH & DELETE add "ProvisioningState" property to the response schema of 200 status code.